Documentation/admin_guide.md

```diff
@@ -39,6 +39,56 @@ If `--wal-dir` flag is set, etcd will write the write ahead log files to the spe
 If you are spinning up multiple clusters for testing it is recommended that you specify a unique initial-cluster-token for the different clusters.
 This can protect you from cluster corruption in case of mis-configuration because two members started with different cluster tokens will refuse members from each other.
 
+#### Monitoring
+
+It is important to monitor your production etcd cluster for healthy information, runtime metrics, and debug variaiables.
+
+##### Health Monitoring
+
+At lowest level, etcd exposes health information via HTTP at `/health` in JSON format. If it returns `{"health": "true"}`, then the cluster is healthy. Please note the `/health` endpoint is still an experimental one as in etcd 2.2.
+
+```
+$ curl -L http://127.0.0.1:2379/health
+
+{"health": "true"}
+```
+
+You can also use etcdctl to check the cluster-wide health information. It will contact all the members of the cluster and collect the health information for you.
+
+```
+$./etcdctl cluster-health 
+member 8211f1d0f64f3269 is healthy: got healthy result from http://127.0.0.1:12379
+member 91bc3c398fb3c146 is healthy: got healthy result from http://127.0.0.1:22379
+member fd422379fda50e48 is healthy: got healthy result from http://127.0.0.1:32379
+cluster is healthy
+```
+
+##### Runtime Metrics
+
+etcd uses [Prometheus](http://prometheus.io/) for metrics reporting in the server. You can read more through the runtime metrics [doc](metrics.md).
+
+##### Debug Variables
+
+Debug variables are exposed for real-time debugging purposes. etcd exposes debug variables via HTTP at `/debug/vars` in JSON format. The debug variables contains
```

Comment:
I would expect some kind of documentation how you might use the data, or how you might consume it. I can curl this, and then I have this giant string..now what?

It seems to compete directly with the prometheus metrics, so I'm not even sure what exactly makes these any better/worse, or why I wouldn't just use the Prometheus metrics for debugging. I mean one has a UI, the other doesn't so what am I getting out of this debug endpoint? It's just not clear how they're different or how I should use them.

After some research it looks like there's Datadog which supports expvars, and https://github.com/divan/expvarmon. Maybe suggest these, and possibly mention what the differences are between Prometheus and debug vars, and when you should use each.